web/themes/custom/liibarrio2020/scss/components/_table-of-contents.scss

```diff
@@ -1,35 +1,18 @@
-.toc-list {
-  padding: 1rem;
-  margin-bottom: 2rem;
-  background-color: $white;
-  border: solid 1px #d5d5d5;
-  list-style-type: none;
-
-  ul,
-  ol {
-    list-style-type: none;
-    padding: 0 0 $spacer-md $spacer-md;
-  }
-
-  li {
-    line-height: 1.25;
-    margin-bottom: 4px;
-
-    &.toc-has-children {
-      margin-bottom: $spacer-md;
-    }
-  }
-
-  & > li > a {
-    font-weight: bold;
-  }
+.toc-container {
+  height: 100vh;
+  overflow: scroll;
+}
 
-  li.toc-has-children > a,
-  .toc-item-doc > a {
-    font-weight: bold;
+la-table-of-contents-controller {
+  .toggle__collapse-all-btn,
```

Comment:
In the spirit of not writing CSS and using classes, like a more naive implementer would do, does it work well if you specify `btn btn-primary` as the classes? Liiweb uses bootstrap after all.

Reply:
cool done